src/pages/components/BookmarkEditDialog.qml

```diff
@@ -23,8 +23,8 @@ UserPrompt {
 
     canAccept: urlField.text && titleField.text && (urlField.text !== url || titleField.text !== title)
 
-    //% "Edit favorite"
-    title: qsTrId("sailfish_browser-he-edit_favorite")
+    //% "Edit"
+    title: qsTrId("sailfish_browser-he-edit")
 
     //: Save the bookmark/favorite
```

Comment:
This could be also cleaned. Something like "Accept text of bookmark edit dialog" as translation help.